index.js

```diff
@@ -156,7 +156,7 @@ module.exports = {
     "array-bracket-spacing": "error",
     "block-spacing": "error",
     "brace-style": "error",
-    "camelcase": "error",
+    camelcase: ["error", { allow: ["^.*_plural$"] }],
```

Comment:
I think we should stay with the double quotes for consistency 😝 

```suggestion
    "camelcase": ["error", { allow: ["^.*_plural$"] }],
```